cdap-docs/developer-manual/source/pipelines/developing-plugins/presentation-plugins.rst

```diff
@@ -687,6 +687,34 @@ CDAP pipelines as of version |version|.
             }
           }
 
+   * - ``toggle``
+     - - ``on``: 'On' state of the toggle, defined by ``value``: value to be set for the plugin property, and
```

Comment:
I think multiple colons in the same sentence is an anti-pattern. Perhaps use ``-`` after ``value`` and ``label`` instead